### PR TITLE
Make FBS and Publizon URLs configurable 

### PR DESCRIPTION
**Pull request by cableman**
#### Link to issue

https://platform.dandigbib.org/issues/5387

#### Description

Make FBS/Publizon mocks available for tests in cypress in actions. 

#### Screenshot of the result

N/A

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

N/A

### CODE REVIEW

File: src/core/utils/helpers/fetcherHelper.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+interface ConfigPossibilities {
+  fbs: string;
+  publizon: string;
+}
+
+export const configTypes: ConfigPossibilities = {
+  fbs: "fbsBaseUrl",
+  publizon: "publizonBaseUrl"
+};
```
Comment (spaceo) [inline src/core/utils/helpers/fetcherHelper.ts:11]:
```suggestion
export const configTypes: ConfigPossibilities = {
  fbs: "fbsBaseUrl",
  publizon: "publizonBaseUrl"
} as const;
```

Comment (spaceo) [reply src/core/utils/helpers/fetcherHelper.ts:11]: If you write `as const` you make sure that the object is immutable and emphasize that it is meant for static "configuration" usage.

File: src/core/publizon/publizonConfig.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
```
Comment (spaceo) [inline src/core/publizon/publizonConfig.ts:1]: This file is deprecated now, right?

File: src/core/fbs/fbsConfig.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/**
```
Comment (spaceo) [inline src/core/fbs/fbsConfig.ts:1]: This file is deprecated now, right?

File: src/core/utils/helpers/fetcherHelper.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { store } from "../../store";
```
Comment (spaceo) [inline src/core/utils/helpers/fetcherHelper.ts:1]: I think you should rename the file to fetcher.ts. That is the naming convention we have used for other helpers.
Alternatively we need to rename all helper files 😈

**Review by spaceo — CHANGES_REQUESTED** [review]
Great. This is more simple.
I have a few comments

File: src/core/publizon/publizonConfig.ts (hunk 2 above)
Comment (sinejespersen) [reply src/core/publizon/publizonConfig.ts]: deleted

File: src/core/fbs/fbsConfig.ts (hunk 3 above)
Comment (sinejespersen) [reply src/core/fbs/fbsConfig.ts]: deleted

File: src/core/utils/helpers/fetcherHelper.ts (hunk 4 above)
Comment (sinejespersen) [reply src/core/utils/helpers/fetcherHelper.ts]: Sure

**Review by spaceo — APPROVED** [review]

**Pull request merged**